3p/ampcontext.js

```diff
@@ -220,6 +220,7 @@ export class AbstractAmpContext {
     this.initialIntersection = context.initialIntersection;
     this.hidden = context.hidden;
     this.sourceUrl = context.sourceUrl;
+    this.configType = context.configType;
```

Comment:
Let's call this remoteConfigType here and throughout and its value can be an enum (right now just NONE and REMOTE_HTML but later will also include REAL_TIME_CONFIG)